main.cpp

```diff
@@ -72,45 +85,83 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
 } 
 
 /*
- 1)
+ 1)Rent a bike
  */
+bool rentBike(double rentDate, int amountOfMoney = 10)
+{
+  ignoreUnused(rentDate, amountOfMoney);
 
+  return{};
+}
 /*
- 2)
+ 2)Play sound
  */
-
+void playSound(float wavFile, float fs = 44100.0f)
+{
+ ignoreUnused(wavFile, fs);
+}
 /*
- 3)
+ 3)Phaser
  */
-
+float phaser(float inputSig, float rate = 0.1f, float depth = 1.0f)
+{ 
+    ignoreUnused(inputSig, rate, depth); 
+    return {}; 
+} 
 /*
- 4)
+ 4)input gain
  */
-
+bool inputGain(float meter, float volume = 1.0f, int style = 3)  
+{ 
+    ignoreUnused(meter, volume, style); 
+    return {}; 
+} 
 /*
- 5)
+ 5)grill meat
  */
-
+void grillMeat(double amountMeat, bool lemon = true)
+{ 
+    ignoreUnused(amountMeat, lemon); 
+} 
 /*
- 6)
+ 6) adress number
  */
-
+int getAdressNumber(int adressNumber)
+{
+  ignoreUnused(adressNumber);
+  return {};
+}
```

Comment:
Please fix the indentation so it conforms with the coding standard.  You'll find the course coding standard in the Readme.md.
- [ ] revise